lib/modules/hover.js

```diff
@@ -118,6 +118,7 @@ class Hover {
 	template = hoverDefaultTemplate;
 	_options = Hover._defaultOptions;
 	_enabled = true;
+	_loadIndicator = '<div class="RESHoverLoadIndicator"><span class="RESLoadingSpinner"></span></div>';
```

Comment:
Probably move this to `HoverInfoCard` or inline it; it's not particularly generic over the class (e.g. I imagine you would want more/different elements if it were to be used for `HoverDropdownList`)